src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -432,6 +447,53 @@ bval example:
 and thus define those values for all sessions and/or subjects in one place (see
 Inheritance principle).
 
+## Multipart (split) DWI schemes
+
+Some MR schemes cannot be acquired directly by some scanner devices,
+requiring to generate several DWI runs that were originally meant to belong
+in a single one.
```

Comment:
I'm not sure if we would want to do this, but would it make sense to use `split` in this case instead of `run`? `run` implies that the parameters are the same, so based on the current definition under **Common principles**, I think you'd want to use `acq` to distinguish the two files (e.g., `acq-first160dirs`, `acq-second160dirs`).

Reply:
At the moment, two `_dwi` runs with all other BIDS entities being equal can differ in there bvecs/bvals specifications. Therefore, you can perfectly have `run-1` and `run-2` encoding the first 160 and the second 160 orientations respectively.

I agree that whether bvecs/bvals should be constant across runs is rather unclear at the moment (and the validator does not check this particular for bvecs/bvals) - I can send a separate PR to clarify this. One thing is sure - there's a lot of data out there with several `run-` indices each with a different gradient table (and I don't think it would be wise more explicitly disallowing this).

That said, the idea for this PR is not to limit the utilization of the proposed new key (`MultipartID`) across runs, also across different acquisitions and/or directions. If the proposal is not clear as regards this, please let me know.

Reply:
@tsalo I have added https://github.com/bids-standard/bids-specification/pull/624/files#diff-d668661006216379ab2a5c5ff7061b8aR450-R452 to refer to this issue more explicitly in the gradient-table section

Reply:
Please let me know further concerns after the latest commits which sideways have contributed to clarify this.

Reply:
I tagged you above, where the exception to the common principles is made explicit. The rest of the discussion I think can be followed with the bulk of the conversation (esp. Chris' comments). Marking resolved - please mark unresolved if you disagree.